scripts/install_rust.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
would prefer like this so it is POSIX compliant and will work with things like zsh so we don't require bash

```suggestion
#!/bin/sh
```